### ISSUE TITLE
Dendrite morphological neural networks

### ISSUE THREAD

**Issue by Ahmad-Masood** [tensorflow/tensorflow#11238]
Can you guys also provide an implementation for Dendrite morphological neural networks in tensorflow. 

http://www.sciencedirect.com/science/article/pii/S0925231213010916

**Comment by gautam1858** [comment]
It would be awesome to work on it, Tensorflow creates the framework to do all these things so we that can work on it

**Comment by sooham** [comment]
Hi, I would like to work on implementing this.

**Comment by MaheshBhosale** [comment]
Yes, I wold also like to contribute to it with guided direction.

**Comment by dynamicwebpaige** [comment]
Though this feature request is outside of the scope of TensorFlow Core, it might make sense to submit it as a PR to [TensorFlow Addons](https://www.github.com/tensorflow/addons). If there is sufficient usage after the component has been migrated to `addons`, we can graduate the feature to `tensorflow/tensorflow`.

Transferring the issue now; @seanpmorgan for visibility.
